package.json

```diff
@@ -60,15 +60,15 @@
   },
   "devDependencies": {
     "@hapi/hapi": "^20.2.1",
-    "nyc": "^15.0.0",
-    "standard": "^16.0.0",
-    "tap-spec": "^5.0.0",
-    "tape": "^5.0.0",
     "@semantic-release/changelog": "^6.0.1",
     "@semantic-release/git": "^10.0.1",
     "commitizen": "^4.2.4",
     "cz-conventional-changelog": "^3.3.0",
-    "semantic-release": "^19.0.3"
+    "nyc": "^15.0.0",
+    "semantic-release": "^19.0.5",
+    "standard": "^16.0.0",
+    "tap-spec": "^2.2.2",
```

Comment:
This shouldn't be downgraded to fix peer issues, there should be a version forward that works with our requirements.